api/lib/domain/models/Organization.js

```diff
@@ -75,6 +75,10 @@ class Organization {
   get isMediationNumerique() {
     return Boolean(this.tags.find((tag) => this.isPro && tag.name === Tag.MEDIATION_NUMERIQUE));
   }
+
+  get isScoAndManagingStudents() {
+    return this.isSco && this.isManagingStudents === true;
```

Comment:
```suggestion
    return this.isSco && this.isManagingStudents;
```

Suggestion: "=== true" pas nécessaire

Reply:
Yeppa je sais, c'était un partis pris pour être explicite et lire ça comme du lait (oui cette expression n'existe pas x))

Reply:
Du coup pourquoi ne pas ajouter "=== true" à this.isSco aussi ? :thinking:

Reply:
Parce que c'est une propriété déjà définie (peut être qu'on voudrait pouvoir la refacto et qu'elle renvoie 'true' un jour 🤔 )

Reply:
Enlever le === true